tests/cupyx_tests/scipy_tests/sparse_tests/test_construct.py

```diff
@@ -420,3 +420,37 @@ def test_kron(self, xp, sp):
 
     # TODO(leofang): check oversize inputs as in scipy/scipy#11879 after
     # #3513 is fixed
+
+
+arrs = [[[0]],
+        [[-1]],
+        [[4]],
+        [[10]],
+        [[1, 2], [3, 4]],
+        [[0, 2], [5, 0]],
+        [[0, 2, -6], [8, 0, 14], [0, 3, 0]],
+        [[5, 4, 4], [1, 0, 0], [6, 0, 8]]]
+
+
+@testing.parameterize(*testing.product({
+    'dtype': (numpy.float32, numpy.float64, numpy.complex64, numpy.complex128),
+    'format': ('csr', 'csc', 'coo'),
+    'arrA': arrs,
+    'arrB': arrs,
```

Comment:
Sorry, one last nit: For both safety and clarity, could you rename `arrs` here to, say, `_arrs_kronsum`, and `arrs` in line 383 etc (for `TestKron`) to `_arrs_kron`? I made a bad choice when I added `TestKron`... Thanks.